src/utility-types.ts

```diff
@@ -37,3 +37,18 @@ export interface MessageValue<
    */
   readonly parent?: MainWorker
 }
+
+/**
+ *
+ */
+export type JustTempValue = {
```

Comment:
Could you please explain why you need that type definition>?

Reply:
We have a new data structure into the pool, the promiseMap ( maybe we can find a better name for that ).
Into this map the key is the messageId of each task submitted, and the value is given by the worker, the resolve function and the reject function.
When we receive a message from the worker we get a map entry and resolve/reject the promise based on the message.
Typescript asked me for types...and I did my best but I need help on that